map.go

```diff
@@ -21,11 +41,31 @@ func HasKey[K comparable, V any](in map[K]V, key K) bool {
 
 // Values creates an array of the map values.
 // Play: https://go.dev/play/p/nnRTQkzQfF6
-func Values[K comparable, V any](in map[K]V) []V {
+func Values[K comparable, V any](in ...map[K]V) []V {
 	result := make([]V, 0, len(in))
 
-	for k := range in {
-		result = append(result, in[k])
+	for i := range in {
+		for k := range in[i] {
+			result = append(result, in[i][k])
+		}
+	}
+
+	return result
+}
+
+// UniqValues creates an array of unique values in the map.
+// Play:
+func UniqValues[K comparable, V comparable](in ...map[K]V) []V {
+	seen := make(map[V]bool)
+	result := make([]V, 0)
+
+	for i := range in {
+		for k := range in[i] {
+			if _, exists := seen[in[i][k]]; !exists {
+				seen[in[i][k]] = true
+				result = append(result, in[i][k])
+			}
+		}
 	}
```

Comment:
Using a temporary variable may improve the readability

```suggestion
	for i := range in {
		for k := range in[i] {{
			key := in[i][k]
			if _, exists := seen[key]; !exists {
				continue
			}
			seen[key] = true
			result = append(result, key)
		}
 	}
```

Please note the remark on map[K]struct{} may apply